### PR TITLE
Add nvidia-apex

### PR DESCRIPTION
**Pull request by oblute**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by oblute** [comment]
@benhuff agree to be a maintainer?

**Comment by oblute** [comment]
@conda-forge/help-python this recipe is ready for review

File: recipes/nvidia-apex/LICENSE [hunk 1]
```diff
@@ -0,0 +1,11 @@
+All rights reserved.
```
Comment (beckermr) [inline recipes/nvidia-apex/LICENSE]: Make sure to make a PR to get them to package this upstream.

File: recipes/nvidia-apex/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - setuptools
+    - pip
+  run:
+    - python
+    - pytorch
+    - cxxfilt >=0.2.0  # [linux]
+    - tqdm >=4.28.1
+    - numpy >=1.15.3
+    - PyYAML >=5.1
+    - pytest >=3.5.1
```
Comment (beckermr) [inline recipes/nvidia-apex/meta.yaml]: These pins need to be as loose as possible. I am skeptical of the tqdm pin in particular. Can you check them?

File: recipes/nvidia-apex/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  imports:
+    - apex
+    - apex.amp
+    - apex.parallel
+    - apex.optimizers
+    - apex.normalization.fused_layer_norm
+
+about:
+  home: "https://nvidia.github.io/apex/"
+  license: BSD 3-Clause
```
Comment (beckermr) [inline recipes/nvidia-apex/meta.yaml]: you should use the SPDX identifier here.

Comment (oblute) [reply recipes/nvidia-apex/meta.yaml]: meaning BSD-3-Clause rather than BSD 3-Clause?

Comment (beckermr) [reply recipes/nvidia-apex/meta.yaml]: yup!

File: recipes/nvidia-apex/LICENSE (hunk 1 above)
Comment (oblute) [reply recipes/nvidia-apex/LICENSE]: @beckermr my bad - it already is packaged in the github repo.

Comment (beckermr) [reply recipes/nvidia-apex/LICENSE]: no worries! simply delete this file and make sure the name of the license matches what is in the recipe!

**Comment by jakirkham** [comment]
Does this need GPU builds? If so, we can discuss next steps in issue ( https://github.com/conda-forge/nvidia-apex-feedstock/issues/1 ).

**Comment by h-vetinari** [comment]
Just found this by accident - it closes #8550. :)

**Pull request merged**